new/chef-workstation.md

```diff
@@ -0,0 +1,131 @@
+   is released, it will likely be renamed to `chef-apply` once the soon-to-be-deprecated
+   command of the same name is removed. The gem containing the `chef-run` command
+   will be renamed `chef-apply`.
+3. The `chef-run` gem may be split into its own repository for future development,
+   at the discretion of the team working on it.
+4. The installer currently called "ChefDK" will be renamed "Chef Workstation".
+   This will require setting up documentation to explain the rename, as well as
+   redirects and supporting pages in the download system to ensure users don't
+   get lost.
+5. The `chef-dk` gem will not be renamed at this time, as few users see the gem
+   name there and we're already changing a lot of things. This may be revisited
+   in the future when the dust settles.
+6. The Omnibus build configuration files and mega-Gemfile/lock will be moved from
+   the `chef-dk` repository to the `chef-workstation` repository (with all the
+   related build job updates that entails). Going forward the `chef-dk` gem/repo
+   will "own" the `chef` CLI tool and the `chef-workstation` repo will down the
+   build process for the installer.
+7. The installer for `chef-dk` (now named "Chef Workstation") will be reconfigured
+   to install into `/opt/chef-workstation` and use `~/.chef-workstation` as the
+   primary configuration folder. A symlink for `/opt/chef-dk` (and related windows-y paths)
+   should be added to ensure any scripts that hardcode paths like `/opt/chefdk/bin/chef`
+   continue to function (until such time as we decide to remove them as compatbility layers).
+   Gems in `~/.chefdk/gems` should be added to the gems path, and config files in
+   `~/.chefdk` should work if no file in `~/.chef-workstation` takes priority.
+8. Add the `chef-apply` gem to the Chef Workstation installer.
+9. A command stub will be added to the `chef-dk` command processor called `apply`,
+   which will dispatch to the `chef-run`/`chef-apply` command. The Chef Workstation installer
+   will map `chef-run` (and later `chef-apply`) into `embedded/bin/` rather than
+   `bin/`. This means that most users will remain unaware of the `chef-run`/`chef-apply` CLI,
+   all documentation will use the `chef apply` command line form. The `chef-run`/`chef-apply`
+   CLI will remain for bleeding edge testers or other complex/advanced use cases.
+
+Additionally, while it is out of scope for this RFC, the authors strongly encourage
+future RFCs and discussions about adding additional tools to Chef Workstation/DK
+that are of value to the Chef community.
+
+## Downsides
+
+This does introduce some potential confusion around the "chef apply"/`chef-apply`
+conceptual space. The existing `chef-apply` command line tool has been useful
+for the very early stages of teaching Chef and for some types of debugging, but
+after those early days, it's mostly ignored. As such, the authors of the RFC feel
+this is an acceptable trade off to make.
+
+## Alternatives
```

Comment:
For the purposes of the eventual merge of this RFC, should this section be renamed to "Alternatives Considered, but not Selected"? Or the contents moved to the pull request as a record of what was considered? I think that would make this RFC more declarative about what is and will be.